submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+* **Nuclear plants require huge amounts of space** - A nuclear setup requires several machines, most of which need to be close to each other. Therefore you need large open areas for them. However, the setups are also considerably energy dense compared to the alternatives, especially on larger scales. In the abovementioned [Reddit post](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/), I found that the 40 MW nuclear setup requires about 1-2 chunks of space, while the equivalent coal power setup requires the same, and the equivalent solar setup requires around ten times as much space. Meanwhile, an 800 MW 2x3 nuclear plant can easily fit into an area of 2x4 chunks thanks to the neighbor bonus and its centrifuges would separately need up to 2 chunks. Therefore you can get twenty times the power for as little as ten times the space, while for coal power and solar power you would flatly need twenty times the space.
+
+* **Nuclear power is a big polluter** - In vanilla Factorio, nuclear radiation causes no damage and has no effects on biter evolution, while regular pollution does. The pollution caused by nuclear power comes only from the mining and processing of uranium ore. I compared this to the pollution from coal power in the same [Reddit post](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/) as above. I found that nuclear power causes only 2-10% of the pollution of mining and burning coal for power.
+
+* **Nuclear reactors are far too expensive to maintain without the Kovarex Enrichment Process** – Obtaining one piece of Uranium-235 indeed requires centrifuging over 1000 uranium ore pieces, but on the other hand a single piece of it is enough to make 10 uranium fuel cells, which last over 33 minutes in total. In a rather eye-opening [tutorial video](https://youtu.be/Qw_NzPuccxk?t=1610), Nilaus walks us through the mathematics, and from there we see that you only need eight electric miners and one centrifuge to maintain one nuclear reactor continuously without the Kovarex Enrichment Process, although it does not hurt to process a little more uranium in preparation for unlocking Kovarex or in case of bad luck during centrifuging. In addition, placing any productivity modules you can get in the centrifuges and fuel assemblers improves your yield significantly.
```

Comment:
```suggestion
* **Nuclear reactors are far too expensive to maintain without the Kovarex enrichment process** – Obtaining one piece of Uranium-235 indeed requires centrifuging over 1000 uranium ore pieces, but on the other hand a single piece of it is enough to make 10 uranium fuel cells, which last over 33 minutes in total. In a rather eye-opening [tutorial video](https://youtu.be/Qw_NzPuccxk?t=1610), Nilaus walks us through the mathematics, and from there we see that you only need eight electric miners and one centrifuge to maintain one nuclear reactor continuously without the Kovarex enrichment process, although it does not hurt to process a little more uranium in preparation for unlocking Kovarex or in case of bad luck during centrifuging. In addition, placing any productivity modules you can get in the centrifuges and fuel assemblers improves your yield significantly.
```